src/lib/core/components/visualizations/PlaceholderIcon.tsx

```diff
@@ -3,7 +3,6 @@ import rxc from './implementations/selectorIcons/RxC.svg';
 import heatmap from './implementations/selectorIcons/heatmap.svg';
 import density from './implementations/selectorIcons/density.svg';
 import map from './implementations/selectorIcons/map.svg';
-import line from './implementations/selectorIcons/line.svg';
```

Comment:
Confused why you removed the icon...?

Reply:
Also the icon is greyed out and not clickable, so I can't do any testing of the viz. From the code I can't immediately see why (looks like you've implemented everything and wired it up), but can look deeper later. Or maybe it's something at my end.
![image](https://user-images.githubusercontent.com/308639/139680413-3568fbd0-6b1e-4b9e-bf5c-f0b3212b1bb8.png)

Reply:
just didnt seem like we needed it there any longer, as theres actually a viz for it to go in now. do we prefer to leave them?

Reply:
Oh, duh, yeah - the clue's in the name!